comixed-tasks/src/main/java/org/comixed/task/model/MoveComicsWorkerTask.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * ComiXed - A digital comic book library management application.
+ * Copyright (C) 2020, The ComiXed Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses>
+ */
+
+package org.comixed.task.model;
+
+import java.util.List;
+import lombok.extern.log4j.Log4j2;
+import org.comixed.model.comic.Comic;
+import org.comixed.repositories.comic.ComicRepository;
+import org.comixed.repositories.tasks.TaskRepository;
+import org.comixed.task.encoders.MoveComicTaskEncoder;
+import org.springframework.beans.factory.ObjectFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Component;
+
+/**
+ * <code>MoveComicsWorkerTask</code> handles encoding instances of {@link MoveComicTask} to move
+ * individual comics.
+ *
+ * @author Darryl L. Pierce
+ */
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+@Log4j2
+public class MoveComicsWorkerTask extends AbstractWorkerTask {
+  private static final int MAX_COMIC_PAGE = 50;
+
+  @Autowired private ObjectFactory<MoveComicTaskEncoder> moveComicWorkerTaskEncoderObjectFactory;
+
+  @Autowired private TaskRepository taskRepository;
+  @Autowired private ComicRepository comicRepository;
+
+  private String directory;
+  private String renamingRule;
+
+  @Override
+  protected String createDescription() {
+    return "Moving comics";
```

Comment:
Couldn't we have a constant string?

Reply:
I think constants are better when the same value is used in more than one spot. If it's a single point, like here, they don't really give any benefit.

Reply:
yes don't give more benefit ok